tests/sdl2_key.c

```diff
@@ -16,7 +16,7 @@ int EventHandler(void *userdata, SDL_Event *event) {
         case SDLK_LEFT: printf("left\n"); result *= 11; break;
         case SDLK_DOWN: printf("down\n"); result *= 13; break;
         case SDLK_UP: printf("up\n"); result *= 17; break;
-        case SDLK_a: printf("a\n"); result *= 19; break;
+        case SDLK_a: break;
```

Comment:
I think it might be better to also verify that this event happened, that is, to modify result here as well (and adjust the final outcome in the test).